services/ingest/tests/test_webhooks_github_app.py

```
@@ -0,0 +1,228 @@
+            "id": run_id,
+            "head_sha": head_sha,
+            "head_branch": "main",
+            "actor": {"login": "octocat"},
+            "head_commit": {"timestamp": "2026-08-01T10:00:00Z"},
+            "run_started_at": "2026-08-01T10:00:00Z",
+            "updated_at": "2026-08-01T10:05:00Z",
+        },
+    }
+
+
+def _deployment_status_payload(installation_id=42, state="success", sha="abc1234567890"):
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `_deployment_status_payload`

(ANN202)